### DISCUSSION TITLE
Cron expression

### DISCUSSION THREAD

**Discussion by codepigai** [airbytehq/airbyte#36379] [Connector Questions]
I write a Cron expression in Airbyte:  0 0 2 10 * ?
means Fire at 2:00am on the 10th day of every month,
but return warning: "10" did not match regex at index,
how can I fix it?
![image](https://github.com/airbytehq/airbyte/assets/128653240/1b88b5db-cd86-465a-8daa-60b87df5ca25)

**Comment by youness-quiet** [comment]
I have the same issue, it works when it's 9th or 11th of the month but not the 10th. Up up

**Comment by nataliekwong** [comment]
Hi there - thanks for reporting this. Looks like a bug. I've created an issue for our team to follow up here: https://github.com/airbytehq/airbyte/issues/44584

**Reply by codepigai** [reply]
thanks for answer, it has been fixed?
open the url,report error:  This issue has been moved to a repository you don't have access to.

**Reply by nataliekwong** [reply]
Thanks for checking in. That move was unintentional and due to some automation on our side. It is now resolved.

You can try a string like this: `0 0 12 10 * ?`
![Screenshot 2024-09-16 at 10 33 02 AM](https://github.com/user-attachments/assets/7ad5ab8c-214d-45d1-8dd3-87b73eac19ee)

**Reply by codepigai** [reply]
thanks,I will plan to do a version update. After that,I will try it 。
